### PR TITLE
MBS-7611 Remove internal documentation

### PR DESCRIPTION
**Pull request by eugene-krivobokov**
Removed pages that have been moved to the internal documentation.

Next steps:
- Migrate remaining pages to mkdocs
- Publish public pages from internal avito documentation

### CODE REVIEW

File: docs/content/menu/index.md [hunk 1]
```diff
@@ -3,74 +3,27 @@ headless: true
 ---
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:3]: Here are all changes in a short way

File: docs/content/menu/index.md [hunk 2]
```diff
@@ -3,74 +3,27 @@ headless: true
...
-    - [Work Manager Testing]({{< ref "/docs/test/WorkManager.md" >}})
     - [Flaky annotation]({{< ref "/docs/test/FlakyAnnotation.md" >}})
-    - [Нестабильные тесты]({{< ref "docs/test/FlakyTests.md" >}})
     - [Toast]({{< ref "/docs/test/Toast.md" >}})
-    - [Snackbar]({{< ref "/docs/test/Snackbar.md" >}})
-    - [Screen checks]({{< ref "/docs/test/ScreenChecks.md" >}})
 - [Test framework]({{< ref "/docs/test_framework/TestFramework.md" >}})
-    - [Mocking]({{< ref "/docs/test_framework/Mocking.md" >}})
+    - [Snackbar]({{< ref "/docs/test_framework/Snackbar.md" >}})
+    - [Screen checks]({{< ref "/docs/test_framework/ScreenChecks.md" >}})
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:20]: Decided to keep pages about the test framework and libraries here.
I want to have them in the repository with the code. Will remove them from internal documentation.

File: docs/content/menu/index.md [hunk 3]
```diff
@@ -3,74 +3,27 @@ headless: true
...
-    - [Statsd]({{< ref "/docs/analytics/Statsd.md" >}})
+    - [Documentation]({{< ref "/docs/contributing/documentation.md" >}})
 - Testing
-    - [Запуск тестов]({{< ref "/docs/test/Run.md" >}})
     - [Test Runner]({{< ref "/docs/test/Runner.md" >}})
-    - [Ручное]({{< ref "/docs/test/Manual.md" >}})
-    - [Test case in code]({{< ref "/docs/test/TestCaseInCode.md" >}})
-    - [Юнит тесты]({{< ref "/docs/test/UnitTesting.md" >}})
-    - [Скриншотные тесты]({{< ref "/docs/test/ScreenshotTesting.md" >}})
-    - [Test minimized]({{< ref "/docs/test/TestMinimized.md" >}})
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:47]: Moved to blog

**Review by dsvoronin — APPROVED** [review]
i still see value in open access to some of the deleted resources

**Comment by eugene-krivobokov** [comment]
> i still see value in open access to some of the deleted resources

Now we have duplicates, the same pages in two places - here and in the internal repository. The goal is to have documentation near the code.
The pages are not deleted and will be accessible publicly later. It's easier to publish from two repositories than support duplicates.

See roadmap in ATBT-7621.

**Pull request merged**